src/Microsoft.ML.StandardLearners/Standard/LogisticRegression/LogisticRegression.cs

```diff
@@ -54,10 +54,24 @@ public sealed class Arguments : ArgumentsBase
         /// <param name="labelColumn">The name of the label column.</param>
         /// <param name="featureColumn">The name of the feature column.</param>
         /// <param name="weightColumn">The name for the example weight column.</param>
+        /// <param name="enforceNoNegativity">Enforce non-negative weights.</param>
+        /// <param name="l1Weight">Weight of L1 regularizer term.</param>
```

Comment:
In most places the comment says "regularization" I think "regularizer" is the naming of an internal object.